### PR TITLE
Updated v2.15.1 docs.

### PR DESCRIPTION
**Pull request by BorisDog**
[New deployment](https://borisdog.github.io/mongo-csharp-driver/)

### CODE REVIEW

File: 2.15/examples/exporting_json/index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
     
     <div>
 <div class="nav-items pull-right">
-  <a href="https://university.mongodb.com" data-toggle="tooltip" data-placement="bottom" title="Try MongoDB in the browser">Try it Out</a>
+  <a href="https://University.mongodb.com" data-toggle="tooltip" data-placement="bottom" title="Try MongoDB in the browser">Learn</a>
```
Comment (JamesKovacs) [inline 2.15/examples/exporting_json/index.html:44]: University in the URL shouldn't be capitalized. Same for additional occurrences below.

File: 2.15/index.xml [hunk 2]
```diff
@@ -191,6 +191,33 @@ not possible to mix use of both modes in the same application.&lt;/p&gt;
...
+&lt;ul&gt;
+&lt;li&gt;BsonBinaryReader.ReadBinaryData method ignores readerSettings.GuidRepresentation&lt;/li&gt;
+&lt;li&gt;BsonBinaryWriter.WriteBinaryData method ignores writerSettings.GuidRepresentation&lt;/li&gt;
+&lt;li&gt;JsonReader ReadBinaryData method ignores readerSettings.GuidRepresentation&lt;/li&gt;
+&lt;li&gt;JsonWriter ignores writerSettings.GuidRepresentation&lt;/li&gt;
+&lt;li&gt;BsonBinaryData ToGuid without GuidRepresentation argument is only valid for sub type 4&lt;/li&gt;
+&lt;/ul&gt;
+</description>
+    </item>
+
```
Comment (JamesKovacs) [inline 2.15/index.xml:220]: Not sure why anything in `2.15/index.xml` is changing in a patch release.

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Overall looks good. Minor changes requested. And take a quick look at why the xml files are different. I don't think they're used, but strange that they were updated by a patch release.

File: 2.15/examples/exporting_json/index.html (hunk 1 above)
Comment (BorisDog) [reply 2.15/examples/exporting_json/index.html]: Fixed.

File: 2.15/index.xml (hunk 2 above)
Comment (BorisDog) [reply 2.15/index.xml:220]: Discussed offline.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**